app/src/main/java/org/fossasia/openevent/app/common/data/repository/EventRepository.java

```diff
@@ -94,4 +99,19 @@ public EventRepository(IUtilModel utilModel, IDatabaseRepository databaseReposit
             .build();
     }
 
+    @NonNull
+    @Override
+    public Observable<Event> updateEvent(Event event) {
+        if (!utilModel.isConnected()) {
+            return Observable.error(new Throwable(Constants.NO_NETWORK));
+        }
+
+        event.setTickets(null);
```

Comment:
I think only distinction should be by date as it is a bad UX to see dividers using 2 criteria